src/vonage/video.py

```diff
@@ -0,0 +1,143 @@
+class Video:
```

Comment:
We need some way to generate a client-side token. While this was handled by `generate_token()` in the old SDK, we should now return a JWT instead.

Reply:
The SDK already has a method for generating JWTs. Inspecting the `generateClientToken` function in the Node SDK shows some differences, so I'll produce a specific method for this task in Python as well.

Reply:
Have added a method to generate JWTs with specific properties, using the Node SDK as a guide.